.vscode/launch.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
@bpbond and @crvernon all of the project files/materials, the files associated with a platform/IDE for hector live in hector/project_files is it a problem that the `.vscode` doesn't live there too? Or does it not really matter because users can't see it since it is a`.dir`?

Reply:
I don't think it matters? But don't know enough about VSCode to opine really.

Reply:
haha I mean I am by no means an expert, but it was a pretty good IDE to download and play around with after about 30 min I was able to build and run hector, much more initiative than Xcode and VS.